metadeploy/consumers.py

```diff
@@ -63,6 +71,16 @@ async def receive_json(self, content, **kwargs):
         # Just used to subscribe to notification channels.
         is_valid = self.is_valid(content)
         is_known_model = self.is_known_model(content.get("model", None))
+
+        # It seems we don't get the correct (updated) value in the session here in the
+        # websocket consumer, after it is set by the scratch_org view in PlanViewSet,
+        # so we accept a value from the user to update the session:
+        uuid = content.get("uuid", None)
+        scratch_org_id = self.scope["session"].get("scratch_org_id", None)
+        if uuid and uuid != scratch_org_id:
+            self.scope["session"]["scratch_org_id"] = uuid
+            self.scope["session"].save()
```

Comment:
@wlonk I'm curious what you think of this solution for the "websocket has out-of-date session" bug?

Reply:
Seems natural, clear, and good.